autoload/lightline/colorscheme/molokai.vim

```diff
@@ -0,0 +1,34 @@
+" =============================================================================
+" Filename: autoload/lightline/colorscheme/molokai.vim
+" Author: challsted
+" License: MIT License
+" =============================================================================
+"
+let s:black = [ '#232526', 233 ]
+let s:gray = [ '#808080', 244 ]
+let s:white = [ '#f8f8f2', 234 ]
+let s:cyan = [ '#66d9ef', 81 ]
+let s:green = [ '#a6e22e', 118 ]
+let s:orange = [ '#ef5939', 166 ]
+let s:pink = [ '#f92672', 161 ]
+let s:red = [ '#ff0000', 160 ]
+let s:yellow = [ '#e6db74', 229 ]
+
+let s:p = {'normal': {}, 'inactive': {}, 'insert': {}, 'replace': {}, 'visual': {}, 'tabline': {}}
+let s:p.insert.left = [ [ s:black, s:green ], [ s:cyan, s:black ] ]
+let s:p.visual.left = [ [ s:black, s:orange ], [ s:black, s:cyan ] ]
+let s:p.replace.left = [ [ s:black, s:red ], [ s:black, s:orange ] ]
+let s:p.normal.left = [ [ s:black, s:pink ], [ s:orange, s:black ] ]
```

Comment:
Would you be opposed to some kind of setting to allow standard and more Molokai like? (Not sure how to word this) in the color scheme would be a if(flag)
Current colors
Else
Standard colors
Endif

Or however color schemes handle this type of logic (i have examples inside of the currunt molokai color scheme. 

This would allow for your standard, but if a user wanted to use more "molokai-like" colors they could. 

If your cool with this, i can do this in here, or in a 2nd pull request